test/unit/domain/WarpFacade.test.ts

```
@@ -191,6 +233,12 @@ describe('v19 Warp facade', () => {
     }
   });
 
+  it('keeps the root export surface to the v19 facade allowlist', () => {
+    const rootExports = exportedNamesFor('index.ts');
+
+    expect([...rootExports].sort()).toEqual([...ALLOWED_ROOT_EXPORTS].sort());
```

ESLint

[error] Use ALLOWED_ROOT_EXPORTS.toSorted() instead of copying and sorting

(e18e/prefer-array-to-sorted)